modules/backend/widgets/Lists.php

```diff
@@ -1154,6 +1155,18 @@ protected function evalCustomListType($type, $record, $column, $value)
         throw new ApplicationException(sprintf('List column type "%s" could not be found.', $type));
     }
 
+    /**
+     * Process as selectable value types for 'dropdown', 'radio', 'checkboxlist', 'balloon-selector' form field types.
+     */
+    protected function evalSelectableTypeValue($record, $column, $value)
+    {
+        $fieldName = $column->columnName;
+        $fieldOptions = $column->config['options'] ?? null;
+        $fieldOptions = $this->getOptionsFromModel($record, $fieldOptions, $fieldName);
+        return $fieldOptions[$value];
```

Comment:
@SaifurRahmanMohsin This should return null if the value does not exist.
```suggestion
        return $fieldOptions[$value] ?? null;
```